src/main/java/uk/gov/hmcts/ccd/AppInsights.java

```diff
@@ -33,6 +37,20 @@ public void trackException(Exception e) {
         telemetry.trackException(e);
     }
 
+    public void trackException(Exception e, Map<String, String> customProperties, SeverityLevel severityLevel) {
```

Comment:
mind adding a javadoc explaining the usage of custom properties

Reply:
Updated to mirror the javadoc comments used in the wrapped Microsoft components.